tests/script/test_add_preload_script.py

```diff
@@ -500,42 +482,47 @@ async def test_preloadScript_add_loadedInNewContexts(websocket, context_id,
 
 
 @pytest.mark.asyncio
-async def test_preloadScript_add_sandbox(websocket, context_id, html):
-    result = await execute_command(
+async def test_preloadScript_add_sandbox_new_context(websocket, html):
+    await execute_command(
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
                 "functionDeclaration": "() => { window.foo='bar'; }",
                 "sandbox": "MY_SANDBOX",
             }
         })
-    assert result == {'script': ANY_UUID}
```

Comment:
`assert result == {'script': ANY_UUID}`

==> keeping this should be harmless

Reply:
It is already tested in a few places, and I removed it to reduce potential noise and improve test readability. Anyways, I rolled it back, if you think it's better to have it.